packages/react-components/src/components/Picker/components/PickerTriggerBody.tsx

```diff
@@ -56,6 +60,30 @@ export const PickerTriggerBody: React.FC<ITriggerBodyProps> = ({
     }
   }, [selectedItems, clearSearchAfterSelection]);
 
+  const onKeyDown = (e: KeyboardEvent) => {
+    if (e.key === 'Backspace' && selectedItems && selectedItems.length > 0) {
+      e.preventDefault();
+      if (type === 'multi') {
+        onItemRemove(selectedItems[selectedItems.length - 1].key);
+      } else {
+        onClear();
+      }
+    }
+
+    if (e.key === 'Delete') {
+      e.preventDefault();
+      onClear();
+    }
+  };
+
+  React.useEffect(() => {
+    if (!isOpen && isTriggerFocused) {
+      document.addEventListener('keydown', onKeyDown);
+
+      return () => document.removeEventListener('keydown', onKeyDown);
```

Comment:
I'm not sure if this hook will always remove added listeners - when a user opens the Picker then the `isOpen` prop will change its value and the removeEventListener will be inaccessible